packages/graphql/src/classes/deprecated/Node.test.ts

```diff
@@ -1,948 +0,0 @@
-/*
```

Comment:
Why this test file is deleted?

Reply:
there's a new one where label and additionalLabels have been replaced with labels.